src/Tribe/Models/Post_Types/Event.php

```diff
@@ -96,34 +116,18 @@ protected function build_properties( $filter ) {
 						/*
 						 * We add one second during this calculation to cope with all-day events starting on 12:00 AM.
 						 * Due to how DateTime diff works diffing two following midnights would yield a diff of 2 days.
+						 * Furthermore, a multi-day event will always last at least 2 days.
 						 */
-						$one_second = new DateInterval( 'PT1S' );
-
-						$this_week_duration = min(
+						$this_week_duration = max( $multiday, min(
 							7,
 							$week_end->diff( $start_date_object->add( $one_second ) )->days + 1,
 							$end_date_object->diff( $week_start )->days + 1,
 							$end_date_object->diff( $start_date_object->add( $one_second ) )->days + 1
-						);
+						) );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: f7ceb900e5186b4363aeb38d48d2dc59</sub>